pages/common/argon2.md

```diff
@@ -0,0 +1,20 @@
+# argon2
+
+> Calculate Argon2 cryptographic hashes.
+> More information: <https://github.com/P-H-C/phc-winner-argon2#command-line-utility>.
```

Comment:
```suggestion
> More information: <https://github.com/P-H-C/phc-winner-argon2>.
```

Reply:
A big chunk of the README is about the hash algorithm itself and the C library and not really about the CLI, so I don't think it's really useful to link to the top of it, since a good amount of scrolling is needed to see the CLI options from there.

Reply:
> A big chunk of the README is about the hash algorithm itself and the C library and not really about the CLI, so I don't think it's really useful to link to the top of it, since a good amount of scrolling is needed to see the CLI options from there.

Agreed, Let's keep it as it is then.